angular/Tile/Overview.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Overview
-description: overview
+title: Overview in Angular Tile Control | Syncfusion
+description: Learn here more about Overview in Syncfusion Essential Angular Tile Control, its elements, and more.
```

Comment:
Change `Overview of` instead of Overview in

Reply:
modified